### ISSUE TITLE
images not displayed consistently

### ISSUE THREAD

**Issue by aha999** [searx/searx#3103]
**Expected behavior**
images should display consistently with all the fields being occupied by photos (unless set differently).

currently it displays some photos and some text, when the category is set to images. I am using Firefox on Android.

**Screenshots & Logs**
![Screenshot_20211224-113510507.jpg](https://user-images.githubusercontent.com/50620416/147345426-c15a0246-7752-4808-9061-1aaf697a45e3.jpg)
![Screenshot_20211224-112954907.jpg](https://user-images.githubusercontent.com/50620416/147345437-8a094298-d6d4-4629-9629-bd0547e46508.jpg)

**Comment by unixfox** [comment]
Which engine are you using?

**Comment by aha999** [comment]
> Which engine are you using?

I am using Firefox on Android.

**Comment by unixfox** [comment]
> > Which engine are you using?
> 
> I am using Firefox on Android.

I'm asking for the engines enabled in the settings of SearX. The search engines.

**Comment by unixfox** [comment]
Closing due to lack of replies

**Comment by aha999** [comment]
> > > Which engine are you using?
> > 
> > I am using Firefox on Android.
> 
> I'm asking for the engines enabled in the settings of SearX. The search engines.

oh, sorry. for pictures - duckduckgo images and google images

**Comment by unixfox** [comment]
It's probably a searx instance issue because I can't reproduce the issue.
Please try with other searx instances from https://searx.space

**Comment by sbstratos79** [comment]
I am hosting my own searx instance and I am facing the same issue. This is also happening with Youtube thumbnails.
![Image 1](https://user-images.githubusercontent.com/4395586/163715944-a721e217-9f3d-4e89-942a-effd5fb87cf6.png)
![Image 2](https://user-images.githubusercontent.com/4395586/163715964-a5357c81-d759-4025-b397-44ad88ddfc5b.png)

EDIT: I did some further testing.
Every image search engine is broken for me. 
`ccengine`, `tineye` and `1x` crash.
Google Images shows about 20 images but the rest of the images are broken.
All the other sources only show broken images.  

EDIT EDIT: I have fixed the issue. It was a problem with my Morty setup. Changing the `result-proxy` url to `[my-domain-name.com]/morty` in settings.yml fixed the issue.
